internal/manifests/queryfrontend/query_frontend.go

```diff
@@ -398,6 +398,11 @@ func enableMonitoringTab(tempo v1alpha1.TempoStack, jaegerQueryContainer corev1.
 			// However, we do not intend to support them.
 			// --prometheus.query.normalize-calls
 			// --prometheus.query.normalize-duration
+			//
+			// NOTE: Jaeger 1.62 default namespace changed to "traces_span_metrics".
+			// We fallback to no namespace.
+			// See https://github.com/jaegertracing/jaeger/pull/6007.
+			"--prometheus.query.namespace=",
```

Comment:
regarding https://github.com/grafana/tempo-operator/issues/1073 (the collector defaults to `traces.span.metrics` namespace we should expose this setting in the CR to allow users using the older collector with Tempo.

Reply:
@frzifus could you please write release notes for 3.4 to reflect the breaking change what we are making here https://github.com/openshift/openshift-docs/pull/83770